Formula/h/hk.rb

```diff
@@ -1,8 +1,8 @@
 class Hk < Formula
   desc "Git hook and pre-commit lint manager"
   homepage "https://hk.jdx.dev"
-  url "https://github.com/jdx/hk/archive/refs/tags/v0.3.3.tar.gz"
-  sha256 "583829ba1c025186bf3dcd79b7e3d3cebd20d4f5a5a1a0cd751366723ba1b812"
+  url "https://github.com/jdx/hk/archive/refs/tags/v0.4.1.tar.gz"
```

Comment:
Can't comment on the right line but `usage` should actually be a runtime dep


```ruby
depends_on "usage"
```

Reply:
yeah, adopted.